### ISSUE TITLE
Parser fails on arithmetic expressions

### ISSUE THREAD

**Issue by vStone** [rodjek/puppet-lint#222]
``` puppet
class foobar {
  $x = 13 % 3
}
```

Parsing:

``` sh
$ puppet parser validate foobar.pp
$ echo $?
0
$ puppet-lint foobar.pp
ERROR: Syntax error (try running `puppet parser validate <file>`) on line 2
```
